.travis.yml

```diff
@@ -8,8 +8,14 @@ services:
 
 script:
   - cd tools/docker
-  - sudo ./image-build.sh -b ubuntu:18.04
+  - sudo ./image-build.sh -b ubuntu:19.04
```

Comment:
Why 19.04?

Reply:
it stil doesn't work, the agent crashes, i'm trying anything I can think of - it certainly not related to our build.. any idea?

Reply:
wasn't related, it appears that our RELEASE is broken on docker container (worked fine locally for me) - see #93